src/xSTIR/pSTIR/STIR.py

```diff
@@ -237,6 +243,76 @@ def get_radii(self):
         ry = _float_par(self.handle, self.name, 'radius_y')
         return (rx, ry)
 
+class SingleScatterSimulator():
+    '''
+    Class for simulating the scatter contribution in an image.
+    This class
+    '''
+    def __init__(self, filename = ''):
+        self.handle = None
+        self.image = None
+        self.name = 'SingleScatterSimulation'
+        self.filename = filename
+
+        if not self.filename:
+            self.handle = pystir.cSTIR_newObject(self.name)
+        else:
+            self.handle = pystir.cSTIR_objectFromFile
+
+        check_status(self.handle)
+
+    def __del__(self):
+        if self.handle is not None:
+            pyiutild.deleteDataHandle(self.handle)
+
+    def set_up(self):
+        print('SingleScatterSimulator:: I am in set up')
+
+    def set_Acquisition_template(self, arg):
+        ''' Set the template of the acquisition data.
+        '''
+        par = pystir.cSTIR_getAcquisitionsData(arg.handle, arg.name);
+        _setParameter(self.handle, self.name, 'template_proj_data_info', par)
+
+    def set_low_energy_threshold(self, threshold):
```

Comment:
should be in `STIRAcquisitionData` (if we really need them)

Reply:
For the time I moved them there